src/_includes/select.html

```diff
@@ -7,4 +7,20 @@
   {% if include.type %}type="{{ include.type }}"{% endif %}
   {% if include.onchange %}onchange="{{ include.onchange }}"{% endif %}
 >
+  {% if include.options %}
+    {% comment %}
+      We can't pass arrays or objects as attributes in an include, so we fall
+      back to a concatenated string.
+      Options are structured like the following:
+      - join the value and the label with a "=="
+      - join the list of value@label with a "|"
+      Eg: "value1==label for value 1|value2==label for value 2"
+    {% endcomment %}
```

Comment:
You might want to include a notice somewhere like readme instead of duplicating code comment? Maintaining comment is probably the thing we will never do.